README.md

```diff
@@ -6,62 +6,40 @@ This repository contains some helper/wrapper scripts to make working with GN eas
 
 ### macOS / Linux
 
-```bash
-git clone https://github.com/MarshallOfSound/electron-gn-scripts.git
-cd electron-gn-scripts
-yarn
-# You could also use `npm install` here
-# You should probably add this to your path in your `.zshrc` or `.bashrc`
-export PATH="$PATH:$(pwd)/nix"
+```sh
+$ git clone https://github.com/MarshallOfSound/electron-gn-scripts.git
+$ cd electron-gn-scripts
+$ export PATH="$PATH:/path/to/electron-gn-scripts/nix"
+# ^ You should probably add this to your `~/.profile`
 ```
 
 ### Windows
 
 ```batch
-git clone https://github.com/MarshallOfSound/electron-gn-scripts.git
-cd electron-gn-scripts
-set PATH="%PATH%;"
-```
-
-## Setup
-
-This toolset does not yet have the ability to initialize an Electron GN setup from scratch so you'll have to
-do the initial work.  These steps are outlined in the [GN Build Instructions](https://github.com/electron/electron/blob/master/docs/development/build-instructions-gn.md) and summarized below.
-
-## Install Depot Tools
-
-You'll need to install [`depot_tools`](https://commondatastorage.googleapis.com/chrome-infra-docs/flat/depot_tools/docs/html/depot_tools_tutorial.html#_setting_up)  to your system.
-
-**On macOS:**
-
-```sh
-# Ensure you're in your home directory
-cd ~
-git clone https://chromium.googlesource.com/chromium/tools/depot_tools.git
+PS> git clone https://github.com/MarshallOfSound/electron-gn-scripts.git
+PS> cd electron-gn-scripts\win
+PS> set PATH="%PATH%;%CD%"
 ```
 
-Add `depot_tools` to the end of your PATH (you will probably want to put this in your `~/.bashrc` or `~/.zshrc`).
-
-**On Windows:**
+On Windows, you'll also need to install [`depot_tools`](https://commondatastorage.googleapis.com/chrome-infra-docs/flat/depot_tools/docs/html/depot_tools_tutorial.html#_setting_up) as outlined in the [GN Build Instructions](https://github.com/electron/electron/blob/master/docs/development/build-instructions-gn.md) and summarized here:
```

Comment:
I've heard rumors that the download of that zip file is **exactly** the same as cloning the git repo 🤔 Might be worth trying

Reply:
I'm not going to have a Windows setup today, so that's going to have to wait for a followup PR